server/database/db_build.sql

```diff
@@ -0,0 +1,45 @@
+BEGIN;
+
+    DROP TABLE IF EXISTS medicine , pharmacy , pharmacist_login , pharmacy_to_medicine
+    CASCADE;
+
+
+CREATE TABLE medicine
+(
+    medicineID serial primary key,
+    name VARCHAR(100) NOT NULL,
+    description VARCHAR(150) NOT NULL,
+)
```

Comment:
Shouldn't there be here a column for `prescription` that is `boolean` according to your prototype ?